source/opt/ir_context.h

```diff
@@ -229,6 +229,8 @@ class IRContext {
   inline void AddExtInstImport(std::unique_ptr<Instruction>&& e);
   // Set the memory model for this module.
   inline void SetMemoryModel(std::unique_ptr<Instruction>&& m);
+  // Get the memory model for this module.
+  inline Instruction* GetMemoryModel();
```

Comment:
nit: shall the `const` equivalent of the function be added instead? (or both).

Reply:
Good point, done